docs/community/community-call.md

```diff
@@ -0,0 +1,52 @@
+---
+id: community-call
+title: Update Call for SAP Community
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Community Call
+description: SAP Cloud SDK hosted community calls to share updates and demonstrate best practices for developers.
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+image:
+---
+
+## Community Call
+
+https://community.sap.com/resources/community-calls
```

Comment:
Let's add a bit of information about what community calls are and why we run them. Then you can give a link to the page where you can understand how to join them. Also, the best practice is to wrap link into a message like: [Folllow this link to register for future community calls](link)